From ba6a3b1fc0cb5e73e2641962a218522fbddf6a2a Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 29 Jun 2022 12:45:23 +0200
Subject: [PATCH] kernel: generic: add missing symbols for RANDOM_TRUST

In kernel 4.14.285, there were added two new config options [1]

- RANDOM_TRUST_CPU [2]
- RANDOM_TRUST_BOOTLOADER [3]

I missed them in my commit 4e7b8f0e0e5c05844d56ee141958208bb4b6b122
("patches/openwrt: kernel: update to version 4.14.285")

[1] https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/drivers/char/Kconfig?h=v4.14.285&id=eb1e322c70cecf656acaeda1f4b52922cc610211

[2] https://cateee.net/lkddb/web-lkddb/RANDOM_TRUST_CPU.html

[3] https://cateee.net/lkddb/web-lkddb/RANDOM_TRUST_BOOTLOADER.html
---
 target/linux/generic/config-4.14 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/linux/generic/config-4.14 b/target/linux/generic/config-4.14
index a0569e4eb0..04a91e3fe4 100644
--- a/target/linux/generic/config-4.14
+++ b/target/linux/generic/config-4.14
@@ -5769,6 +5769,8 @@ CONFIG_XFRM=y
 # CONFIG_XILINX_WATCHDOG is not set
 # CONFIG_XILINX_ZYNQMP_DMA is not set
 # CONFIG_XILLYBUS is not set
+# CONFIG_RANDOM_TRUST_CPU is not set
+# CONFIG_RANDOM_TRUST_BOOTLOADER is not set
 # CONFIG_XIP_KERNEL is not set
 # CONFIG_XMON is not set
 CONFIG_XZ_DEC=y
-- 
2.34.1

